server/server.go

```diff
@@ -1,113 +1,52 @@
 package server
 
-import (
-	"path/filepath"
-
-	pb "github.com/kubernetes/kubernetes/pkg/kubelet/api/v1alpha1/runtime"
-	"github.com/opencontainers/ocitools/generate"
-	"golang.org/x/net/context"
+const (
+	runtimeAPIVersion = "v1alpha1"
```

Comment:
@hmeng-19 https://github.com/kubernetes/kubernetes/pull/17048/files#r71423073

In our case, it can be either left blank, or we can use ocid version (?)